### PR TITLE
Icon Font Updated

### PR DESCRIPTION
**Pull request by harpalsingh**
# Description

Updated Icon Font to be aligned further with DLS and added all required Icons that were still in SVG format into the Icon Font itself and remove all SVGs for Icons.

# Visuals

All Icons in the Icon Font now:

Screenshot 2019-09-05 at 15 31 54

# Testing Instructions

Ensure using the Icons that were SVGs still work using the Icon Font.

Icons to check in particular:

- in_transit
- arrow_left_small
- arrow_right_small
- progress
- bin
- basket
- credit_card_slash
- plus_large
- minus_large
- remove
- grid
- location
- coins

### CODE REVIEW

File: src/components/icon/icon.style.js [hunk 1]
```diff
@@ -108,7 +86,7 @@ const StyledIcon = styled.span`
       border-radius: ${iconSizeConfig.backgroundShape[bgShape]};
     `}
 
-    ${isFont && css`
+    ${css`
```
Comment (edleeks87) [inline src/components/icon/icon.style.js]: you can remove the `${css ...}` from around the `&::before styles here

File: src/components/icon/icon-classic.style.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ export default ({
     }
   `}
 
-  ${isFont && css`
+  ${css`
```
Comment (edleeks87) [inline src/components/icon/icon-classic.style.js]: You don't need to wrap the `&::before in the the `${css...}`

File: src/components/icon/icon.style.js (hunk 1 above)
Comment (edleeks87) [reply src/components/icon/icon.style.js]:
```suggestion
    &::before {
      font-size: 16px;
      font-style: normal;
      font-weight: normal;
      line-height: 16px;
      vertical-align: middle;
      ${type === 'services' && browserTypeCheck(window) && css`margin-top: -7px;`}
      ${type === 'services' && isSafari(navigator) && !browserTypeCheck(window) && css`margin-top: -4px;`}
    }
```

**Comment by DlgSHi** [comment]
During the test session I've noticed few things
- Warning - is to low for Small bgTheme
- Submited - lack of the icon
- Individual  - lack of the icon
- White tick - lack of the icon

**Pull request merged**